internal/packager/fork.go

```diff
@@ -51,10 +57,12 @@ func Fork(originName, forkName, outputDir string, maintainers []string) error {
 				return err
 			}
 		}
-		dest := filepath.Join(appPath, k)
-		log.Debugf("Writing file at %s", dest)
-		if err := ioutil.WriteFile(dest, v, 0644); err != nil {
-			return errors.Wrap(err, "error writing output file")
+
+		// Create the directories for any nested files
+		basepath := filepath.Dir(fullFilepath)
+		os.MkdirAll(basepath, os.ModePerm)
+		if err := ioutil.WriteFile(fullFilepath, v, 0644); err != nil {
+			return errors.Wrap(err, "failed to write output file:"+fullFilepath)
```

Comment:
nit: you can use `errors.Wrapf(err, "failed to write output file: %s", fullFilepath)`